find.sh

```diff
@@ -12,4 +14,5 @@ for i in "supplied.corepop" `ls -1r ${OSNAME}/${ARCH}/*.corepop`; do
     fi 
 done
 # If no good executable can be found, exit with an error code.
+>&2 echo "No valid corepop found."
```

Comment:
Because the output is so verbose, I wonder if we need to make that message standout a bit more?

Reply:
Yeah?
It currently does this:

```
make build
make[1]: Entering directory '/home/will/projects/getpoplog/Seed'
mkdir -p "_build"
cp -rpP _download/Base/ _build/
make -C _build/Base build
make[2]: Entering directory '/home/will/projects/getpoplog/Seed/_build/Base'
mkdir -p _build
echo 'void test(){}' > _build/test.c
if `(cd _build; /usr/bin/gcc -no-pie -c test.c 2>&1)`; then make buildNoPie; else make buildWithoutNoPie; fi
make[3]: Entering directory '/home/will/projects/getpoplog/Seed/_build/Base'
test ! -z "-v -no-pie -Wl,-export-dynamic -Wl,-no-as-needed" # Protect against being called from the wrong context.
sed -e 's/{{{POP__CC_OPTIONS:[^}]*}}}/-v -no-pie -Wl,-export-dynamic -Wl,-no-as-needed/' < pop/src/syscomp/x86_64/asmout.p.template > pop/src/syscomp/x86_64/asmout.p
true
make[3]: Leaving directory '/home/will/projects/getpoplog/Seed/_build/Base'
make[2]: Leaving directory '/home/will/projects/getpoplog/Seed/_build/Base'
mkdir -p _build/poplog_base
( cd _build/Base; tar cf - pop ) | ( cd _build/poplog_base; tar xf - )
touch _build/Base.proxy # Create the proxy file to signal that we are done.
cp -rpP _download/Corepops _build/
/bin/rm -rf _build/Corepops/linux/x86_64/*
cp -p _build/poplog_base/pop/pop/corepop _build/Corepops/supplied.corepop
make -C _build/Corepops corepop
make[2]: Entering directory '/home/will/projects/getpoplog/Seed/_build/Corepops'
COREPOP=`./find.sh` && [ ! -z "$COREPOP" ] && cp -p "$COREPOP" corepop
No valid corepop found.
make[2]: *** [Makefile:20: corepop] Error 1
make[2]: Leaving directory '/home/will/projects/getpoplog/Seed/_build/Corepops'
make[1]: *** [Makefile:464: _build/Corepops.proxy] Error 2
make[1]: Leaving directory '/home/will/projects/getpoplog/Seed'
```

I suppose it's not that loud, I'll add something that makes it read like this:
```
##################################
# ERROR: No valid corepop found. #
##################################
```